src/main/java/org/folio/dew/batch/bulkedit/jobs/updatejob/BulkEditUpdateUserRecordsJobConfig.java

```diff
@@ -0,0 +1,103 @@
+package org.folio.dew.batch.bulkedit.jobs.updatejob;
+
+import org.folio.dew.domain.dto.User;
+import org.folio.dew.domain.dto.UserFormat;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.JobBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import org.springframework.batch.core.launch.support.RunIdIncrementer;
+import org.springframework.batch.item.ItemProcessor;
+import org.springframework.batch.item.ItemReader;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.batch.item.file.FlatFileItemReader;
+import org.springframework.batch.item.file.LineMapper;
+import org.springframework.batch.item.file.builder.FlatFileItemReaderBuilder;
+import org.springframework.batch.item.file.mapping.BeanWrapperFieldSetMapper;
+import org.springframework.batch.item.file.mapping.DefaultLineMapper;
+import org.springframework.batch.item.file.mapping.FieldSetMapper;
+import org.springframework.batch.item.file.transform.DelimitedLineTokenizer;
+import org.springframework.batch.item.file.transform.LineTokenizer;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.FileSystemResource;
+
+import static org.folio.dew.domain.dto.EntityType.USER;
+import static org.folio.dew.domain.dto.ExportType.BULK_EDIT_UPDATE;
+import static org.folio.dew.utils.Constants.FILE_NAME;
+
+@Configuration
+public class BulkEditUpdateUserRecordsJobConfig {
+
+  @Bean
+  public Job bulkEditUpdateUserRecordsJob(
+    Step bulkEditUpdateRecordsStep,
+    JobBuilderFactory jobBuilderFactory) {
+    return jobBuilderFactory
+      .get(BULK_EDIT_UPDATE.getValue() + "-" + USER.getValue())
```

Comment:
In UploadController bulk edit jobs are filtered by exportType.getValue().equals(job.getName()) only so "-" + USER.getValue() is not necessary

Reply:
Since in the future there will be other jobs for updating other types of records I've put the type of entities with which the job works.